### PR TITLE
Add coverage

### PR DESCRIPTION
**Pull request by sseraj**
## Purpose
Added codecov coverage

## Type of change
What types of change is it?
_Select the appropriate type(s) that describe this PR_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (non-backwards-compatible fix or feature)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Documentation update
- [x] Maintenance update
- [ ] Other (please describe)

### CODE REVIEW

**Comment by nwu63** [comment]
Okay I think the coverage is 0 because we are running `subprocess.run` which spawns a subprocess, meaning `coverage.py` can't really track what's going on. Perhaps this is the time to actually convert examples to tests so we run them within the same python process?

**Comment by sseraj** [comment]
I just tried running a basic test locally and can confirm that the issue is with `subprocess`. I'll work on converting the tests. Should I make a separate PR? This will also require some changes to cgnsUtilities.

**Comment by marcomangano** [comment]
Yeah I think a separate PR makes sense, then we just merge `master` here and hope that everything works as it should

**Comment by anilyil** [comment]
Is this ready for review or are we waiting for #54?

File: .github/build_real.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@
 set -e
 cp $CONFIG_FILE config/config.mk
 make
-pip install .
+pip install -e .
```
Comment (nwu63) [inline .github/build_real.sh:5]: This is no longer needed

**Review by nwu63 — APPROVED** [review]

**Review by anilyil — APPROVED** [review]

**Pull request merged**